articles/setup-ssh-ubuntu-vm-aws/index.md

```diff
@@ -0,0 +1,120 @@
+
+```bash
+mv your_pem_file.pem ~/.ssh/your_pem_file.pem
+```
+
+Change from your current directory to the `~/.ssh` directory where you placed the private key file:
+
+```bash
+cd ~/.ssh
+```
+
+Next, you need to ensure your private key file (your_pem_file.pem) is not publicly viewable for SSH to work. To do this, run:
+
+```bash
+chmod 400 ubuntu.pem
+```
+
+To gain an understanding of how the `chmod` command works, read up on [Linux File Permissions](https://www.linux.com/training-tutorials/understanding-linux-file-permissions/).
+
+At this point, you're almost ready to SSH into your Virtual Machine Instance. You need one last thing – your Virtual Machine's IP address. You can get this from the AWS console, as shown in the image below.
+![Virtual Machine IP address](/engineering-education/setup-ssh-ubuntu-vm-aws/ip_address.png)<br>
+Now you can ssh into your instance by running the ssh command:
+
+```bash
+ssh -i ~/.ssh/your_pem_file.pem ubuntu@your_virtual_machine_ip_address -v
+```
+
+You will then get a response like:
+
+```bash
+The authenticity of host '13.244.115.171 (13.244.115.171)' can't be established.
+ECDSA key fingerprint is SHA256:hTaJnzw/oDXxzLCMHcp9ieHcHxEISfxubiEkylIhtkc.
+Are you sure you want to continue connecting (yes/no)?
+```
+
+Enter `yes`.
+
+And You DID IT!!! You have successfully used SSH to get remote access to your Virtual Machine on AWS. Feel free to poke around!!
+
+To better understand the ssh command you used above, see the breakdown below.
+
+1. **ssh**: the ssh command is used to start the SSH client program that enables secure connection to the SSH server on a remote machine.
+2. **-i**: identity_file, a file from which the [identity key](https://www.ssh.com/ssh/identity-key) (private key) for [public key authentication](https://www.ssh.com/ssh/public-key-authentication) is read.
+3. **"~/.ssh/your_pem_file.pem"**: the [absolute path](https://www.linux.com/training-tutorials/absolute-path-vs-relative-path-linuxunix/) to your_pem_file.pem
+4. **ubuntu**: the username you will use to ssh into the virtual machine.
+5. **@**: used to refer the virtual machine IP address much like @gmail denotes a Gmail email servers.
+6. **your_virtual_machine_ip_address**: used to denote the IP address you wish to SSH into.
+7. **-v**: verbose mode, where every action is printed to the screen as it happens.
+
+For more information read [SSH Command](https://www.ssh.com/ssh/command).
+
+When you're done, exit the SSH connection by running:
+
+```bash
+logout
+```
+
+*Note:* Remember to terminate the instance to avoid charges when the trial period ends.
+
+To terminate the instance, right click on the instance in the console:
+
+**Instance State > Terminate**
+
+![Terminate Virtual Machine Instance](/engineering-education/setup-ssh-ubuntu-vm-aws/terminate_instance.png)
```

Comment:
Might be worth cross referencing this with other articles.  Perhaps the same concept, setting up an aws instance, via terraform, if we have one of those or maybe this is just a future consideration.  @ninjaginja